python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -461,78 +478,84 @@ def _annotate_image(fig, source, flags):
 
     flag_color = "red"
     text_color = "grey"
+
+    if len_sizes == 1:
+        heights = [0.95, 0.91, 0.87, 0.83, 0.79]
+    else:
+        heights = [1.2, 1.15, 1.1, 1.05, 1.0]
```

Comment:
Did you check this for more than 2 cutouts? I'd bet making 3 cutouts would need a new set of values, too. I'm still skeptical that supporting more than 2 cutouts is worth the bother: I don't see what it gains, and it adds complexity in several places.

Pulling those values out into an array is a good idea: I should have done that in the first place! Sorry it made changing this a pain.

Reply:
yes, I tried this with up to 5 cutouts and it works well. The flags/metadata are clearly visible with the cutouts looking aligned as expected.